conda_rpms/install.py

```diff
@@ -579,13 +834,15 @@ def remove_binstar_tokens(url):
             meta_dict['files'] = list(yield_lines(alt_files_path))
             os.unlink(alt_files_path)
         except IOError:
-            meta_dict['files'] = files
+            meta_dict['files'] = all_files
         meta_dict['link'] = {'source': source_dir,
                              'type': link_name_map.get(linktype)}
         if 'channel' in meta_dict:
             meta_dict['channel'] = remove_binstar_tokens(meta_dict['channel'])
         if 'icon' in meta_dict:
             meta_dict['icondata'] = read_icondata(source_dir)
+        if noarch:
+            meta_dict['noarch'] = noarch
```

Comment:
@lbdreyer The `meta_dict` is used below by the function `create_meta`, which essentially writes the associated `index.json` for the package into the `conda-meta` directory.

However, if `noarch is not False` then it must be the case that `index.json` already contains a `noarch` entry. So surely there is no need to add it again here ... because `noarch` can only be `not False` if `noarch` is already in the existing `index.json`.

Reply:
yep, I'll remove this